provider/oci/environ.go

```diff
@@ -95,6 +107,10 @@ func (e *Environ) allInstances(tags map[string]string) ([]*ociInstance, error) {
 	return ret, nil
 }
 
+func (e *Environ) SetClock(clock clock.Clock) {
+	e.clock = clock
```

Comment:
Do we need this in regular testing?
Usually we expose this via either a constructor, or only in "export_test.go" so it ends up available to the test suite, but not generally available to consumers.

Reply:
Ping? If we only need this in the test suite, and we don't need it outside of *this* test suite, I'd rather this just ends up in export_test.go

Reply:
removed